### PR TITLE
Barplot of number of detected feature per sample

### PR DESCRIPTION
**Pull request by zitoa**
Please review the barplot of number of detected features per sample. We should be able to show it tomorrow. Thanks.

### CODE REVIEW

**Review by ESCRI11 — CHANGES_REQUESTED** [review]
Couple of points for improvement

File: components/board.dataview/R/dataview_plot_totalcounts.R [hunk 1]
```diff
@@ -50,79 +59,107 @@ dataview_plot_totalcounts_server <- function(id,
...
-        plotly_default() %>%
-        plotly::layout(
-          xaxis = list(title = FALSE),
-          yaxis = list(title = res$ylab),
-          margin = list(l = 30, r = 0, t = 0, b = 0)
-        )
-
-      fig
+      
+      if(res$sampleqc_plottype == "Average total abundance") {
```
[the remark this answers is not in the document]
Comment (zitoa) [reply components/board.dataview/R/dataview_plot_totalcounts.R:134]: Thank you Xavi

File: components/board.dataview/R/dataview_plot_totalcounts.R [hunk 2]
```diff
@@ -50,79 +59,107 @@ dataview_plot_totalcounts_server <- function(id,
         ("")
       }
 
-      ylab <- paste0("total ", type, logtype)
-      if (data_groupby != "<ungrouped>") {
-        ylab <- paste0("average total ", type, logtype)
+      sampleqc_plottype <- input$sampleqc_plottype
+
+      ## ylab <- paste0("total ", type, logtype)
```
[the remark this answers is not in the document]
Comment (zitoa) [reply components/board.dataview/R/dataview_plot_totalcounts.R:64]: Thank you Xavi - it was just for my reference i will delete it

File: components/board.dataview/R/dataview_ui.R [hunk 3]
```diff
@@ -195,9 +195,10 @@ DataViewUI <- function(id) {
           dataview_plot_totalcounts_ui(
             ns("counts_total"),
             label = "a",
-            title = "Total counts",
+            title = "Total counts / Number of detected features",
             info.text = "Average total counts by sample. Samples can be grouped using the {Group by} setting.",
```
[the remark this answers is not in the document]
Comment (zitoa) [reply components/board.dataview/R/dataview_ui.R:199]: will do thanks

**Pull request merged**